src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const obj = { ...state };
+  const arr = [];
+
+  for (const key of actions) {
+    if (key.type === 'addProperties') {
```

Comment:
try use switch case when you want to check a value for multiple possible results